zuul.d/jobs.yaml

```diff
@@ -682,6 +682,28 @@
       - Arch:x86_64
       - BuildType:Integration test
 
+- job:
```

Comment:
So if we don't need to test with k8s v1.13 any more? Maybe we should delete the job 1.13 definition above in jobs.yaml?

Reply:
It's not easy to remove them since many CPO branches  contains them in .zuul.yaml. We should remove them from CPO side first.

Reply:
Actually, I tried that , it didnt allow me because this job is being used for other branches eg: on release1.15 / release1.14 still this job runs.

Reply:
OK, let's leave it as it is now.